scripts/azure/run_azure.sh

```diff
@@ -74,9 +75,7 @@ git clone https://github.com/charmed-hpc/charmed-hpc-benchmarks.git
 cd charmed-hpc-benchmarks
 
 # Recursively run all checks
-reframe -C config/azure_config.py -c checks -r -R
-
-logout
+reframe -C config/azure_config.py -c checks -r -R -S slurmrestd_api_check.slurmrestd_hostname="$1"
```

Comment:
[nit]: Since this a script and not something we need to interactively type into the shell, could we use the long form of the flags here if they exist. It will make things much more clear to anyone who is running these benchmarks but aren't super familiar with the intricacies of ReFrame.